src/chrome/content/rules/mk2.xml

```diff
@@ -0,0 +1,12 @@
+<ruleset name="MK2">
+  <target host="mk2.com" />
+  <target host="www.mk2.com" />
+  <target host="m.mk2.com" />
```

Comment:
Please also add api.mk2.com and comments for refused media.mk2.com and mismatched preprod.mk2.com